store/__tests__/snackbar.spec.ts

```diff
@@ -25,49 +25,51 @@ beforeEach(() => {
   store = new Vuex.Store(cloneDeep({ state, mutations, actions }));
 });
 
-test('When "SET_SNACKBAR_MESSAGE" is committed, the snackbar message is set.', () => {
-  store.replaceState({ ...defaultState() });
+describe('vuex: snackbar', () => {
+  it('sets the snackbard message when "SET_SNACKBAR_MESSAGE" is committed', () => {
```

Comment:
Snackbard lmao, I'm thinking of a snackbar with a beard and some luth or something